pvlib/atmosphere.py

```diff
@@ -97,25 +97,21 @@ def alt2pres(altitude):
 
 def get_absolute_airmass(airmass_relative, pressure=101325.):
     '''
-    Determine absolute (pressure corrected) airmass from relative
-    airmass and pressure
+    Determine absolute (pressure-adjusted) airmass from relative
+    airmass and pressure.
 
-    Gives the airmass for locations not at sea-level (i.e. not at
-    standard pressure). The input argument "AMrelative" is the relative
-    airmass. The input argument "pressure" is the pressure (in Pascals)
-    at the location of interest and must be greater than 0. The
-    calculation for absolute airmass is
+    The calculation for absolute airmass (AMa) is
 
     .. math::
-        absolute airmass = (relative airmass)*pressure/101325
+        airmass_absolute = airmass_relative \times \frac{pressure}{101325}
```

Comment:
* use raw string or double escape LaTeX math, see comment above
* I prefer implicit multiplication, `\times` looks like a cross product
* Use curly braces around subscript
```
airmass_{absolute}=airmass_{relative}\frac{pressure}{101325}
```
![absolute airmass](https://latex.codecogs.com/png.latex?airmass_{absolute}=airmass_{relative}\frac{pressure}{101325})

IMO verbose variable names are great for code, but don't look good in typeset math equations, I prefer:
```
AM_a=AM_r\frac{p}{101325}
```
![absolute airmass](https://latex.codecogs.com/png.latex?AM_a=AM_r\frac{p}{101325})

Follow with definitions of each variable,
```
where :math:`AM_a` is absolute airmass ...
```